### PR TITLE
📝. EG Specification: Key Generation

### LINKED ISSUE #135: 📝. Document Key Generation Section of 0.95.0 Specification in Markdown
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Suggestion

Document **Key Generation** Section of 0.95.0 Specification in Markdown for easy consumption on the web. This should be as close as possible to a 1 to 1 translation.

### Possible Implementation

Using the 0.95.0 Specification (https://github.com/microsoft/electionguard/releases/download/v0.95.0/ElectionGuard_Specification_v0_95_0.pdf) provided, create a markdown documentation page for the specific section mentioned. The page should be created within the `spec/0.95.0 folder`

#### Headings
There should be only one H1 heading which will presented as the title of the page.
Any other headings should be H2 or smaller according to their section. 

#### Equations
Equations should use the built-in feature.
https://squidfunk.github.io/mkdocs-material/reference/mathjax/

#### Tables
Tables should use the built-in feature.
https://squidfunk.github.io/mkdocs-material/reference/data-tables/

#### Images
We want to keep the images matching the existing style of the site. These should not be copy and pasted from the document, but created preferably. If there is no other option, screenshots from the document will be accepted. 

The images may be created as svg's using https://themeisle.com/illustrations/ and/or https://undraw.co/ with the color (#80CBC4) (Preferred)

OR

By using mermaid documentation: 
https://mermaid-js.github.io/mermaid/#/
https://squidfunk.github.io/mkdocs-material/reference/diagrams/

#### Green and Blue Boxes

All blue boxes should be Note admonitions.
All green boxes should be Important admonitions
https://squidfunk.github.io/mkdocs-material/reference/admonitions

#### Navigation

This is the overall navigation
```
Current or 0.95.0 <-- Overview 
--- Structure <-- ElectionGuard Structure
--- Components <-- ElectionGuard Components (2nd ElectionGuard Components)
--- Baseline Parameters
--- Key Generation
--- Ballot Encryption
--- Ballot Aggregation
--- Verifiable Decryption
--- Election Record <-- The Election Record
--- Verifier Construction
------ Parameter Validation
------ Guardian Public-Key Validation 
------ ...
--- Applications <-- Applications to end-end verifiability and risk limited audits
--- Acknowledgements 
```

This specific page should be linked under both 0.95.0 and Current in the `mkdocs.yml`.
An example is https://www.electionguard.vote/spec/0.95.0/1_Overview/

#### Pull Request​​ Requests
Any pull request should specifically tag @benaloh and @keithrfung as reviewers. The pull request MUST include copious screenshots.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Sumithra-Sudhakar**
[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
 

Fixes #135 

### Description
*Key generation part of the document is converted to markdown*

### CODE REVIEW

**Comment by keithrfung** [comment]
@Sumithra-Sudhakar The spacing is very specific. I think the documentation may be slightly off. See the below PR that has an example

https://github.com/microsoft/electionguard/pull/147/files

**Comment by Sumithra-Sudhakar** [comment]
I have made some changes pls check

**Comment by keithrfung** [comment]
Marking this as hacktoberfest approved ahead of the weekend, but will review on detail on Monday.

**Review by keithrfung — COMMENTED** [review]
There is still a bunch of empty space but I think that's a nitpick.

Approval from me but @benaloh has final approval.

File: docs/spec/0.95.0/5_Key_Generation.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+ # Key Generation
```
Comment (keithrfung) [inline docs/spec/0.95.0/5_Key_Generation.md:1]: There is an extra space here. 

` # Key Generation` -> `# Key Generation`

**Review by keithrfung — CHANGES_REQUESTED** [review]

**Review by benaloh — CHANGES_REQUESTED** [review]
Many of the formulae seem to have not rendered in the screenshots.  It would be much easier to evaluate if the formulae are fully rendered.

One additional note is that the Guardian indices T_1, T_2, ..., T_n should be subscripted but seem not to be.

Thanks.

**Comment by keithrfung** [comment]
@Sumithra-Sudhakar  To ease the pain of the screenshots, you may notice we added Netlify for previews. This will ideally simplify the review process. 

I am going to let @benaloh review.

**Comment by benaloh** [comment]
Thanks for all of the work on his.  I'm seeing lots of issues within the variable and formulae that need to be corrected -- especially associated with with subscripts, superscripts, and spacing.  For instance, variable like T1, T2, ..., Tn and Ki should all be subscripted.  A subscript of "ij" is different from "i,j".  Also there are many places where there is extra space between variable and their subscripts or too little space between distinct variables.  Also the long display equation starting with g^{P_i(\alpha)} contains many errors that need to be fixed, and a product should always be represented with a capital pi rather than a lower case pi -- although there is probably an explicit product symbol that should be used.

Thanks.

**Comment by keithrfung** [comment]
@Sumithra-Sudhakar Thanks for all the work. Since you haven't followed up on Josh's recent comments, going to close this PR. Feel free to reopen and revisit when you're ready.

**Pull request closed without merging**